models/apis/check_collisions_test.go

```diff
@@ -0,0 +1,33 @@
+package apis
```

Comment:
we probably want a build tag here so this test cannot run outside of our environment since its reaching out to git cli.

maybe use `awsinclude` for the build tag, maybe `codegen`, but awsinclude seems to fit better.